WORKSPACE

```diff
@@ -165,8 +166,13 @@ maven_install(
         "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.2.2",
         "org.jetbrains:annotations:jar:13.0",
         "org.mockito:mockito-core:2.19.0",
+        "org.mockito:mockito-core:2.7.22",
         "org.robolectric:annotations:4.3",
         "org.robolectric:robolectric:4.3",
+        "androidx.navigation:navigation-fragment:2.0.0", #
```

Comment:
Why we need `#` at the end?

Reply:
We dont, removed them and alphabetized them :)